### ISSUE TITLE
Cumulative Bandwidth, IOPS, Latency.

### ISSUE THREAD

**Issue by jo-ale35** [axboe/fio#584]
Problem Statement: Would like to have cumulative output line for min, max and mean bandwidth, iops and latency values. 

As of now the summary line has the a single cumulative bandwith and iops result which I belive is the cumulative of bw_mean and iops_mean. We don't have this for latency.  

How do you calculate the bandwidth and iops in the summary line?
How can we calculate it for latency as well.

**Comment by sitsofe** [comment]
@jo-ale35 I don't think you mean cumulative (i.e. since the job started), I think you mean "only for I/Os done in the last second / last interval" (moving/running/rolling average) - is this right?

If so this is a duplicate of https://github.com/axboe/fio/issues/130 ...

**Comment by sitsofe** [comment]
@jo-ale35 does it sound like #130 describes the feature you're after?

**Comment by jo-ale35** [comment]
Not exactly. In #130 the requirement is to get the results in each second or in a given time interval. But what I want is latency value in the summary data that we get in the fio output which now gives Bandwidth and IOPS

**Comment by jo-ale35** [comment]
It will be appreciated if we can get the bandwidth and iops results in a certain given interval.

**Comment by sitsofe** [comment]
@jo-ale35 So if I understand correctly over an interval you want:
* Interval's bandwidth
* Interval's IOPS
* Interval's latency

?
